packages/liveblocks-core/src/crdts/LiveList.ts

```diff
@@ -134,7 +134,16 @@ export class LiveList<TItem extends Lson> extends AbstractCrdt {
     ops.push(op);
 
     for (const item of this._items) {
-      ops.push(...item._toOps(this._id, item._getParentKeyOrThrow(), pool));
+      const parentKey = item._getParentKeyOrThrow();
+      const childOps = HACK_addIntentAndDeletedIdToOperation(
+        item._toOps(this._id, parentKey, pool),
+        undefined
+      );
+      const childOpId = childOps[0].opId;
+      if (childOpId !== undefined) {
+        this._unacknowledgedSets.set(parentKey, childOpId);
```

Comment:
Out of scope for this change, but I want to flag a slight worry here that it doesn't feel obvious that (or why) you need to mutate the `_unacknowledgedSets` structure after turning this into a set op above. Or mainly that it's now _this function's_ responsibility to do that. After looking at other code, I can understand it's the right thing, but the rationale isn't too obvious from the code yet, and I think how we update `_unacknowledgedSets` in various places may not be abstracted well-enough. It's mostly just very easy to make a mistake here — e.g. by forgetting to update this set.

But again, definitely out of scope for this PR.